docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -138,6 +138,14 @@ Note that the `TabularDataset` object is essentially a [pandas DataFrame](https:
 
 To solely use custom data preprocessing and automatically apply your custom transformations to both the train data and all future data encountered during inference, you should instead create a custom FeatureGenerator. Follow this example in the source code: [examples/tabular/example_custom_feature_generator.py](https://github.com/awslabs/autogluon/blob/master/examples/tabular/example_custom_feature_generator.py)
 
+### I'm receiving C++ warning spam during training or inference
```

Comment:
add the exact warning message here, the point of this is so when someone googles the warning, it will link them here. (This is what most people do)

Reply:
Yeah, good point. Added this.